src/applications/disability-benefits/all-claims/pages/medicalCare.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
```

Comment:
Is there something more specific we can name this page to avoid confusion with potential other forms within the all-claims flow? Maybe `unemployabilityMedicalCare`? I know it's quite verbose/long but I think it's better than getting this mixed up with other medicalCare pages in the future.